src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                + numbers.toString()));
```

Comment:
```suggestion
                + numbers));
```
toString() is redundant here

Reply:
Renamed

Reply:
Removed